packages/generator-single-spa/src/react/templates/package.json

```diff
@@ -22,7 +22,6 @@
     "@babel/runtime": "^7.8.7",
     "@testing-library/jest-dom": "^5.5.0",
     "@testing-library/react": "^9.4.0",
-    "@types/jest": "^24.0.23",
```

Comment:
I actually like having `@types/jest` and `@types/systemjs` installed for non typescript projects, because it provides autocomplete when i do `describe()` and `System.import()` inside of the IDE. VS Code will try to autocomplete those things incorrectly otherwise.

I understand that that's a weird reason to have a dependency, though, so I'm okay with removing this and suffering through worse autocomplete :)

Reply:
That makes total sense! Should that be for both `jest` and `systemjs` types?